avaframe/out3Plot/plotUtils.py

```diff
@@ -191,19 +182,119 @@
 cmapDEM2 = {}
 cmapDEM2['cmap'] = cmapGreys
 cmapDEM2['colors'] = colorsS
-cmapDEM2['lev'] = None
-cmapDEM2['ticks'] = None
+cmapDEM2['levels'] = None
 
 cmapAimec = cmapAvaframe
 
 cmapVar = {}
-cmapVar['cmap'] = cmapAvaframeCont
+# cmapVar['cmap'] = cmapAvaframeCont
 cmapVar['colors'] = colorAvaframe
-cmapVar['lev'] = None
-cmapVar['ticks'] = None
+cmapVar['levels'] = None
 
 cmapGreys1 = {}
 cmapGreys1['cmap'] = cmapGreys
+
+
+def makeColorMap(colormapDict, levMin, levMax, continuous=False):
+    """Get colormap for plot
+
+    get the colormap, norm, levels... for ploting results
+
+    colormapDict is a dict as described in the Parameters section or a matplotlib
+    cmap. In the second case, the continuous flag is ignored and a continuous cmap is returned with
+    levelsNew and colorsNew set to None.
+    In the first case, depending on the continuous flag and what is given in the colormapDict
+    dictionary, the cmap is created and the levelsNew and colorsNew are created.
```

Comment:
suggestion: 
Options: 
1. colormapDict = dict (details see Parameters) - depending on the continuous flag and what is given in the colormapDict dictionary, the cmap is created and the levelsNew and colorsNew are created.
2. colormapDict = matplotlib cmap - the continuous flag is ignored and a continuous cmap is returned with
    levelsNew and colorsNew set to None.